fab/operations/supervisor.py

```diff
@@ -253,9 +253,12 @@ def _format_env(current_env, extra=None):
         ret['supervisor_env_vars']['NEW_RELIC_CONFIG_FILE'] = '%(root)s/newrelic.ini' % env
         ret['supervisor_env_vars']['NEW_RELIC_ENVIRONMENT'] = '%(environment)s' % env
 
+    all_hosts = [host.name for host in InventoryParser(env.inventory).groups['all'].get_hosts()]
+
     if env.http_proxy:
         ret['supervisor_env_vars']['http_proxy'] = 'http://{}'.format(env.http_proxy)
         ret['supervisor_env_vars']['https_proxy'] = 'https://{}'.format(env.http_proxy)
+        ret['supervisor_env_vars']['no_proxy'] = '{},relay.nic.in,smsgw.sms.gov.in,icds-cas.gov.in'.format(','.join(all_hosts))
```

Comment:
do you think we can put these extra hosts in a variable in `environments.yml`? something like `additional_no_proxy_hosts` maybe?